app/pages/returns/ExportedHumanMedicinesPlasticPackagingPage.scala

```diff
@@ -31,7 +31,7 @@ case object HumanMedicinesPlasticPackagingPage extends QuestionPage[Boolean] {
   override def cleanup(value: Option[Boolean], userAnswers: UserAnswers): Try[UserAnswers] = {
```

Comment:
do we have any unit test for this?

Reply:
Not yet. Journey up in the air so will need to come back to this.